README.md

```diff
@@ -1,6 +1,6 @@
 # dp-legacy-cache-proxy
 
-Proxy for handling the cache for pages within the legacy CMS
+The proxy handles the cache for pages within the legacy CMS. It receives requests from dp-frontend-router and redirects them to Babbage with the correct cache header. The cache header is calculated based on the information retrieved from dp-legacy-cache-api.
```

Comment:
> cache header

`Cache-Control` header. It would also be worth mentioning that it does not do this for all pages.

> The cache header is calculated based on the information retrieved from dp-legacy-cache-api.

This is not right, there are many instances in which the Proxy does not need to contact the API.

I have updated Confluence with some high level information on how the Proxy and the API work, it may be worth reading them for some inspiration.